kuryr_kubernetes/controller/managers/prometheus_exporter.py

```diff
@@ -134,3 +142,13 @@ def _create_metrics(self):
         self.pod_creation_latency = prometheus_client.Histogram(
             'kuryr_pod_creation_latency', 'Time taken for a pod to have'
             ' Kuryr annotations set', buckets=buckets, registry=self.registry)
+
+        self.load_balancer_readyness = prometheus_client.Enum(
+            'kuryr_load_balancer_readyness',
+            'Indicates issues with creating Octavia LB',
+            states=['ok', 'notready'], registry=self.registry)
```

Comment:
Should the 'ok' stauts be ever set? or perhaps the metric type should be a Gauge?

Reply:
+1, we need a strategy to get the problematic status cleared up.

Reply:
During creation of the Enum type, the state is automatically set to the first one from the list.
As for status reset - after the kuryr controller restart it should "reset" itself. Unless we have some other way for cleaning failed port/lb, I can't see any other option for removing an alert.